index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+        <meta content="text" charset="utf-8">
+        <link rel="stylesheet" href="styles.css">
+        <title>Kiplaacollo Portfolio website</title>
+
+    </head>
+    <body>
+        <header class="head">
+                <h3 class="logo">Welcome</h3>
```

Comment:
- [ ] Kindly I recommend you to use `a` tag instead of `h3` tag for logo , This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.